src/borg/fuse.py

```diff
@@ -340,7 +340,7 @@ def read(self, fh, offset, size):
                     # evict fully read chunk from cache
                     del self.data_cache[id]
             else:
-                data = self.key.decrypt(id, self.repository.get(id))
+                data = self.key.decrypt(id, self.repository_uncached.get(id))
```

Comment:
Yes for small cache sizes (<< memory) the VFS will likely be doing a good job. For cache sizes >> memory there are use cases where this would still be useful. I would expect mounting block device / image backups via a borg mount -> loopback setup would likely be cases where is make a significant difference. We could open a ticket where we can discuss if we care. Because this PR is of course already merged.